src/newlayout/components/Button/Cooldown.tsx

```diff
@@ -1,26 +1,31 @@
 import { useLoading } from "@/stores/loadingStore";
-import { shallowEqual } from "@/utils/helpers";
+import { useEventType } from "@/stores/mapStore";
+import { EVENT_TYPES } from "@/types";
 import { Button, LinearProgress, SxProps, Theme } from "@mui/material";
 import { useTranslation } from "next-i18next";
-import { useRouter } from "next/router";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { mutate } from "swr";
 
 interface IStyles {
   [key: string]: SxProps<Theme>;
 }
 
+const scanAreaWhiteList: Partial<EVENT_TYPES[]> = ["moveend", "zoomend"];
+
 export const CooldownButtonComponent = () => {
   const { t } = useTranslation("home");
-  const router = useRouter();
   const { loading } = useLoading();
-  const [queryParams, setQueryParams] = useState<{
-    [key: string]: string | string[] | undefined;
-  }>({ ...router.query });
-  const isSameParams = shallowEqual(router.query, queryParams);
+  const eventType = useEventType();
+  const [isDisabled, setIsDisabled] = useState(false);
+
+  useEffect(() => {
+    if (!scanAreaWhiteList.includes(eventType)) return;
```

Comment:
let's rename this to `allowList`, terms like the one we are using are marked as offensive: https://www.theregister.com/2019/09/03/chromium_microsoft_offensive/

Reply:
ok, I'll fix this ASAP. Thanks for the information